### PR TITLE
Five more UGens

### PR DESCRIPTION
**Pull request by kureta**

### CODE REVIEW

File: supriya/tools/ugentools/Duty.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # -*- encoding: utf-8 -*-
 from supriya.tools.ugentools.UGen import UGen
-
+from supriya import *
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/ugentools/Duty.py]: Could you remove this import?

File: supriya/tools/ugentools/Duty.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 class Duty(UGen):
-    r'''
+    r'''A value is demanded of each UGen in the list and output according to a stream of duration values.
 
     ::
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/ugentools/Duty.py]: For the `Duty` example, build it off of an example from the SC documentation to make the usage more obvious:

```
duty = ugentools.Duty.kr(
    done_action=0,
    duration=ugentools.Drand(
        sequence=[0.01, 0.2, 0.4],
        repeats=2,
        ),
    reset=0,
    level=ugentools.Dseq(
        sequence=[204, 400, 201, 502, 300, 200],
        repeats=2,
        ),
    )
```

File: supriya/tools/ugentools/Duty.py [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+        'level',
+        'done_action',
+    )
+
+    _valid_calculation_rates = None
+
+    # ### INITIALIZER ###
+
+    def __init__(
+            self,
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/ugentools/Duty.py]: Why two levels of indentation here?

File: supriya/tools/ugentools/Duty.py [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+            calculation_rate=None,
+            done_action=0,
+            duration=1,
+            level=1,
+            reset=0,
+    ):
```
Comment (josiah-wolf-oberholtzer) [inline supriya/tools/ugentools/Duty.py]: While I'd prefer a hanging indent (this line indented four more spaces), I'm OK with this.

In the future, I may try to run Google's `yapf` Python formatter against the codebase. That code beautifier can't do the hanging-indent style I prefer, and would output this de-dented parens style instead. Oh well...

**Comment by kureta** [comment]
This should be formatted as you want.

File: supriya/tools/ugentools/Duty.py (hunk 3 above)
Comment (kureta) [inline supriya/tools/ugentools/Duty.py]: I let PyCharm format the code amd reverted some changes by hand. So I guess the answer is it is too late and  I should sleep :)

I'll look into yapf in the morning. I would really like to have  a tool autoformat my code and have a consistent style without worrying about it too much.

Comment (josiah-wolf-oberholtzer) [inline supriya/tools/ugentools/Duty.py]: `yapf` does some things well, and other things poorly. I don't know that Python has a "hands-free" formatting solution.

Luckily, all of the code in `pendingugentools` was auto-generated, so it doesn't need to be reformatted when finishing up a UGen's implementation.

**Pull request merged**